src/components/Navbar/index.spec.tsx

```diff
@@ -23,18 +23,23 @@ import Navbar from '.';
 describe(__filename, () => {
   type RenderParams = {
     _requestLogOut?: typeof requestLogOut;
+    reviewersHost?: string;
```

Comment:
This must be an older test file. I suggest changing it to `RenderParams = Partial<PublicProps> & Partial<DefaultProps>`.